pkg/kritis/signer/signer.go

```diff
@@ -61,24 +57,7 @@ var (
 	authFetcher = authority.Authority
 )
 
-// ValidateAndSign validates image from vulnz signing policy and then creates
-// attestation for the passing image.
-// Returns an error if image does not pass or creating an attestation fails.
-func (s Signer) ValidateAndSign(imageVulnz ImageVulnerabilities, vps v1beta1.VulnzSigningPolicy) error {
-	glog.Infof("Validating %q against VulnzSigningPolicy %q", imageVulnz.ImageRef, vps.Name)
-	violations, err := s.config.Validate(vps, imageVulnz.ImageRef, imageVulnz.Vulnerabilities)
-	if err != nil {
-		return fmt.Errorf("error when evaluating image %q against policy %q", imageVulnz.ImageRef, vps.Name)
-	}
-	if violations != nil && len(violations) != 0 {
-		return fmt.Errorf("image %q does not pass VulnzSigningPolicy %q: %v", imageVulnz.ImageRef, vps.Name, violations)
-	}
-
-	glog.Infof("Image %q passes VulnzSigningPolicy %s.", imageVulnz.ImageRef, vps.Name)
-	return s.SignImage(imageVulnz.ImageRef)
-}
-
-// ValidateAndSign signs an image without doing any policy check.
+// SignImage signs an image without doing any policy check.
 // Returns an error if image does not pass or creating an attestation fails.
```

Comment:
Update comment: there's nothing to pass.

Reply:
Done.